Add documentation for action icons and action divider (#185)
avo-hq/avodocs: docs/3.0/customizable-controls.md

# Or
actions_list exclude: [ExportSelection, PublishPost]
```
::::info

> The actions icon and action divider will defined in def action. Here is an example of [action icon and divider](actions.md#action_icons_and_action_divider)

Reviewer: ```suggestion
The actions icon and dividers are defined in `def actions` method. Here is an example of [action icon and divider](actions.md#action_icons_and_action_divider)
```
